packages/go/stbernard/command/builder/builder.go

```diff
@@ -69,6 +71,13 @@ func (s *command) Parse(cmdIndex int) error {
 	if err := cmd.Parse(os.Args[cmdIndex+1:]); err != nil {
 		cmd.Usage()
 		return fmt.Errorf("parsing %s command: %w", Name, err)
+	} else {
+		if *targetOs != "" {
+			s.env.Override("GOOS", *targetOs)
+		}
+		if *targetArch != "" {
+			s.env.Override("GOARCH", *targetArch)
+		}
```

Comment:
These don't really need to be in the else clause, I'd just remove the else and let them be floating after the error check, since they have zero dependencies on that error check.